roles/foreman_devel/tasks/main.yml

```diff
@@ -1,3 +1,27 @@
 ---
 - include_tasks: github_push_ssh.yml
   when: foreman_devel_github_push_ssh
+
+- include_tasks: check_forks_of_repos.yml
+  when: katello_devel_github_username is defined and foreman_installer_options_internal_use_only is defined
+
+- name: "Check situation, if the machine is behind NAT"
+  # for example OpenStack instances
+  # This is potential problem, if user has the same private IP in own lan. 
+  shell: "ping -c 1 {{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}"
+  delegate_to: localhost
+  ignore_errors: True
+  register: rping
+
+- name: "Add OS hostname to hosts file" 
+  lineinfile:
+    dest: /etc/hosts
+    regexp: ".*{{ ansible_fqdn }}.*$"
+    line: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }} {{ ansible_fqdn }}"
+    state: present
+  become: true
+  when: "rping.rc != 0"
+
+- include_tasks: rhel.yml
+  when: ansible_distribution == "RedHat"
```

Comment:
Doesn't CentOS fall under this distribution family? In which case, the actions in the task wouldn't make as much sense.

Reply:
Centos has got these values:
  "ansible_distribution": "CentOS",
  "ansible_distribution_file_variety": "RedHat", 
  "ansible_distribution_major_version": "7", 
  "ansible_distribution_release": "Core", 
  "ansible_distribution_version": "7.2.1511",